### PR TITLE
Compress the test data

### PR DESCRIPTION
**Pull request by nicholasbishop**
tl;dr: The test data is now compressed, shrinking the file to about 1.6MiB instead of 64MiB.

Shrinking the test data is helpful because its stored via Git LFS, and Github charges somewhat aggressively for LFS bandwidth. CI jobs that download LFS blobs are counted against the bandwidth limit, even though it's presumably very cheap for the vendor. Rather than give in to this rather greedy scheme, add a little bit of code complexity and decrease the amount of data being stored.

The reason for implementing our own scheme, rather than some standard compression such as lz4, is to minimize dependencies in ext4-view. Even though the decompression code is only needed in tests, and therefore only needs to be a dev-dependency, users sometimes have to do extra work to vet or import dependencies, and these requirements don't always exempt dev-dependencies. Also, it's fun.

### CODE REVIEW

**Comment by tedbrandston** [comment]
I feel like this might not be a good idea. As you say: "users sometimes have to do extra work to vet or import dependencies" and if I were vetting this I'd be nervous if I saw that the build system used a custom compression scheme to download binary blobs. It just sounds... sketchy.

**Comment by nicholasbishop** [comment]
That's an interesting angle. I wouldn't think it's a problem since the code isn't shipped with the library; on the decompression code is present, and that only in tests. `xtask` is just used in the repo for development. Does that change your opinion at all?

**Comment by tedbrandston** [comment]
I don't know that I have a strong opinion. I haven't done much vetting of dependencies, so I'm probably not the best person to weigh in on it. It seems like this might be a tradeoff between number of deps and ease of vetting? 

I'll review it more thoroughly.

**Comment by nicholasbishop** [comment]
Moving to draft while I think about alternatives

**Comment by nicholasbishop** [comment]
Closing in favor of https://github.com/nicholasbishop/ext4-view-rs/pull/221
